### PR TITLE
This commit adds SSL-support for filebeat-input

### PR DESCRIPTION
**Pull request by ralfbosz**

### CODE REVIEW

**Comment by ralfbosz** [comment]
Need to add SSL-configuration to filebeat::input

https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-syslog.html#filebeat-input-syslog-tcp-ssl

**Comment by cdenneen** [comment]
@ralfbosz please submit fix for this or revert until fixed. Thanks

File: templates/input.yml.erb [hunk 1]
```diff
@@ -198,6 +198,11 @@
     <%- end -%>
   <%- end -%>
   fields_under_root: <%= @fields_under_root %>
+  ssl:
+    <%- @ssl.each_pair do |k, v| -%>
+    <%= k %>: <%= v %>
+    <%- end -%>
+  <%- end -%>
```
Comment (pcfens) [inline templates/input.yml.erb:205]: Mirroring @cdenneen's comment here - https://github.com/pcfens/puppet-filebeat/commit/21d02c597e0cdc900f7515eceeee35e8a28a1c90#r61848537

**Comment by pcfens** [comment]
I suspect that the extra end block was related to a conditional that was missed (if someone doesn't specify any ssl parameters we probably don't need that section at all).

Once the right fix is in I'm happy to merge again.

Thanks @cdenneen for finding the issue so quickly, though it does make me wonder - are you deploying this module from GitHub instead of the forge? :laughing:

**Comment by cdenneen** [comment]
@pcfens i have users pulling master for their fixtures for unit testing.

**Pull request merged**